peldd.cc

```diff
@@ -296,44 +296,61 @@ struct Arguments {
     "/usr/i686-w64-mingw32/sys-root/mingw/bin"
   }};
   unordered_set<string> whitelist;
-  const array<const char*, 36> default_whitelist = {{
+  const array<const char*, 53> default_whitelist = {{
```

Comment:
If you're using C++17 now, you can do away with this evil hardcoded integer by using the deduction guide provided for `std::array`:
https://en.cppreference.com/w/cpp/container/array/deduction_guides

Reply:
Unfortunately using C++17 is optional, so instead of deduction guides we will just use `std::vector` with initializer lists to remove the hardcoded count.